Dockerfile

```diff
@@ -38,26 +39,26 @@ RUN yarn install --check-files
 COPY . .
 
 RUN bundle exec rake assets:precompile && \
-    rm -rf tmp/* log/* node_modules /tmp/*
+    rm -rf tmp/* log/* /tmp/*
```

Comment:
Sorry @vassyz, can you just remind me again why we've removed `node_modules` from here?

Reply:
I mentioned it in the description, sort of. `puppeteer`(with all its dependencies) runs when generating files from the `node_modules` folder, so if you remove it, it will crash when you try to download a pdf.